gtfs-realtime/spec/en/trip-modifications.md

```diff
@@ -50,7 +52,7 @@ Each `ReplacementStop` message defines a stop that will now be visited by the tr
 
 When `travel_time_to_stop` is specified, the `arrival_time` is calculated from a reference stop in the original trip, plus the offset in `travel_time_to_stop`. Otherwise, the `arrival_time` can be be interpolated based on the total duration of the modification in the original trip.
 
-The `departure_time` always equals the `arrival_time`.
+The dwell time can be defined with the `dwell_time` field, and by default it's 0.
```

Comment:
The dwell_time description seems inconsistent. In a proto comment it says "If not specified, the same dwell time from the schedule is applied." But in the reference table they say "If not specified, the dwell time is assumed to be 0, and the arrival time and departure time will be the same" and here "The dwell time can be defined with the `dwell_time` field, and by default it's 0". 

Can you clarify?

Reply:
Thanks for finding that error

The correct definition is 
> The dwell time can be defined with the dwell_time field, and by default it's 0. 

I had put same as schedule initially but it doesn't make sense since it's for replacement stop and those don't have schedule.